### ISSUE TITLE
[Improvement] On-screen feedback for how many frames the player is in an idle state

### ISSUE THREAD

**Issue by rsnoob35** [Grouflon/3rd_training_lua#32]
Where an idle state is defined as an animation that can be cancelled by normal attacks.

I can think of several instances where this would be useful information to have: to practice tight blockstrings, to practice acting quickly after an opponent's attack, and to practice precisely delayed options after getting hit/blocking/getting knocked down

If it's difficult to get the data for every frame of every idle state, the first 15-20 would probably be the most important/most used

**Comment by Grouflon** [comment]
I'm not sure what you mean by idle state here. For me, an idle state is by definition always cancellable by a normal or special attack. Can you develop ?

**Comment by rsnoob35** [comment]
Yep, that's what I mean. The states that would be included would probably be walking, crouching, standing still, and jumping
